migrations/7_deploy_creditor_proxy.js

```diff
@@ -0,0 +1,46 @@
+const CONSTANTS = require('./migration_constants');
```

Comment:
We recently merged in another PR that had a 7th migration, so you can rename this to `migrations/7_deploy_creditor_proxy.js` 😄 

Might be best to merge master into this branch so you can test it.